assets/src/scss/pages/post/_comments-block.scss

```diff
@@ -55,6 +55,8 @@
     .comment-reply-link {
       float: right;
       margin-top: 0;
+      color: var(--comment-reply-link--color, var(--link--color));
```

Comment:
Wouldn't these styles automatically be applied if we merge the changes for [this ticket](https://jira.greenpeace.org/browse/PLANET-6990)? 🤔

Reply:
It make sense your comment!

Reply:
👍 Updated!